tests/reg_tests/test_adjoint.py

```diff
@@ -174,8 +174,68 @@ def span(val, geo):
     },
 ]
 
+# Adding this separately because we don't have the complex version available yet
```

Comment:
Are you able to add a complex test now that ANK is complex-safe? The test you added is good, but it doesn't prove that the bug has been fixed.

Reply:
This is a good point, initially it was supposed to be added in a second moment... but probably it is better to not procrastinate anymore